modules/archetypes/lib/policy_set_definitions/policy_set_definition_es_enforce_encrypttransit.tmpl.json

```diff
@@ -490,7 +490,7 @@
       },
       {
         "policyDefinitionReferenceId": "PostgreSQLEnableSSLDeployEffect",
-        "policyDefinitionId": "${root_scope_resource_id}/providers/Microsoft.Authorization/policyDefinitions/Deploy-PostgreSQL-sslEnforcement",
+        "policyDefinitionId": "${root_scope_resource_id}/providers/Microsoft.Authorization/policyDefinitions/Deploy-PostgreSql-sslEnforcement",
```

Comment:
Need to fix name at source in [Azure/Enterprise-Scale](https://www.github.com/Azure/Enterprise-Scale)